src/pages/Login.jsx

```diff
@@ -1,6 +1,90 @@
+import { TextField } from "../components"
+import { useState} from "react";
+import axios from "axios";
+import { useNavigate } from 'react-router';
+import { useAuth } from "../contexts";
+import { Link } from "react-router-dom";
+
 const Login = () => {
+
+  const [loginData, setLoginData] = useState({
+    email: "",
+    password: ""
+  })
+
+  const [loginErrorMsg, setLoginErrorMsg] = useState("");
+  const {isLoggedIn,setIsLoggedIn, userDetails, setUserDetails} = useAuth();
+  const navigate = useNavigate();
+  const testCredentials = {
+    email: "piyushdas98@gmail.com",
+    password: "Piyush@123"
+  }
+
+  const handleChange = (e) => {
+    const {name,value} = e.target;
+    setLoginData((prev)=>({...prev, [name]: value}))
+  }
+
+  const handleLogin = async(credentials) => {
+    try {
+      const response = await axios.post("/api/auth/login",credentials);
+      if(response.status===200 || response.status===201){
+      setIsLoggedIn(true);
+      setLoginErrorMsg("");
+      const {data} = response;
+      const {firstName,cart,wishlist} = data.foundUser;
+      setUserDetails({
+        cartList: cart,
+        wishList: wishlist,
+        firstName: firstName
+      });
+      localStorage.setItem("userToken",data.encodedToken)
+      navigate("/products")
+    }
+      else{
+        if(response.status=404)
```

Comment:
here you are not checking the status, you are assigning 404 to status?

Reply:
Ohh. Thank you so much for pointing this out.
I dont know I missed it. Will correct it :D